src/main/scala/org/dbpedia/spotlight/wikistats/RawWikiMain.scala

```diff
@@ -34,9 +34,9 @@ object RawWikiMain {
     val lang = args(1)
 
     val sparkConf = new SparkConf()
-      .setMaster("local[5]")
+      //.setMaster("local[5]")
```

Comment:
These comments were added just to distinguish between local and cluster mode. I will remove now.